AmplicaLabs/social-app-template
# 73 UI cleanup and flavor

# Purpose
The goal of this PR is clean up the UI of the SAT frontend

Closes #73 

## Solution
Reformat and improve the UI of the SAT

@p5150j @mattheworris 


### Change summary
* Too many to list


## Steps to Verify
1. Run the app locally and poke around
2. If a bug is encountered, check that it's not already recorded in the SAT Projects Board.

## Additional details / screenshot
- 
<img width="1239" alt="Screenshot 2024-07-11 at 4 01 04 PM" src="https://github.com/user-attachments/assets/bff1d214-2ca0-4af0-b353-10ceba024c77">
